loadtest/user/userentity/scheduled_posts.go

```diff
@@ -0,0 +1,76 @@
+package userentity
+
+import (
+	"context"
+	"github.com/mattermost/mattermost/server/public/model"
+)
+
+func (ue *UserEntity) CreateScheduledPost(teamId string, scheduledPost *model.ScheduledPost) error {
+	user, err := ue.getUserFromStore()
+	if err != nil {
+		return err
+	}
+
+	scheduledPost.UserId = user.Id
+	createdScheduledPost, _, err := ue.client.CreateScheduledPost(context.Background(), scheduledPost)
+	if err != nil {
+		return err
+	}
+
+	err = ue.store.SetScheduledPost(teamId, createdScheduledPost)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (ue *UserEntity) UpdateScheduledPost(teamId string, scheduledPost *model.ScheduledPost) error {
+	user, err := ue.getUserFromStore()
+	if err != nil {
+		return err
+	}
+
+	scheduledPost.UserId = user.Id
+	updatedScheduledPost, _, err := ue.client.UpdateScheduledPost(context.Background(), scheduledPost)
+	if err != nil {
+		return err
+	}
+
+	err = ue.store.SetScheduledPost(teamId, updatedScheduledPost)
+	if err != nil {
+		return err
+	}
+
+	ue.Store().UpdateScheduledPost(teamId, updatedScheduledPost)
```

Comment:
Do we need to set it and then update it? Also, we should check the error from UpdateScheduledPost

Reply:
Just update should be fine, fixed this.